requirements/default.txt

```diff
@@ -209,6 +209,9 @@ MySQL-python==1.2.3c1
 # sha256: sboSwcRTI_2Glb5oznIoRWuG5RI08WcCfrvEzNqFLGE
 nose==1.1.2
 
+# sha256: REZqm8tW0uVodQ-RUE0SeMdOq7JZowWwbpdbh7UWNdo
+nose-exclude==0.4.1
```

Comment:
Sure, I originally put it alongside nose, as if nose is present and in use, we probably want to control the exclusion of the pytests.